infrastructure/azure.tf

```diff
@@ -1023,6 +1026,62 @@ resource "azurerm_function_app" "azurerm_function_app_app" {
   ]
 }
 
+resource "azurerm_function_app" "azurerm_function_app_public" {
+  name                      = "${local.azurerm_functionapp_public_name}"
+  location                  = "${azurerm_resource_group.azurerm_resource_group.location}"
+  resource_group_name       = "${azurerm_resource_group.azurerm_resource_group.name}"
+  app_service_plan_id       = "${azurerm_app_service_plan.azurerm_app_service_plan.id}"
+  storage_connection_string = "${azurerm_storage_account.azurerm_functionapp_storage_account.primary_connection_string}"
+  client_affinity_enabled   = false
+  version                   = "~2"
+
+  https_only                = true
+  enable_builtin_logging    = true
+
+  site_config = {
+    # We don't want the express server to idle
+    # so do not set `alwaysOn: false` in production
+    always_on = true
```

Comment:
this depends on the plan, they made some changes since this was set to "true", I'll double check with @LucaPrete and @cloudify  if this is still needed (as it impacts costs)

Reply:
All the changes done by @LucaPrete are in the new infrastructure. The app service plan in the old AGID infrastructure has not been changed. I am missing something?

Reply:
no, it's ok if is still the app service plan (and NOT the consumption one)

Reply:
This PR https://github.com/teamdigitale/io-infrastructure/pull/34 change the plan to the consumption one but i have verified that this change was not applied using terraform so the service plan running is still the Standard one.